<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1526</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="campo_grafico3" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>691</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="info_senal_ica">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>601</width>
     <height>641</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_graficar_ica">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Graficar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
